### ISSUE TITLE
Default <select> Value?

### ISSUE THREAD

**Issue by winrid** [adamghill/django-unicorn#546]
Hi Adam :) First - let me say Unicorn is an awesome idea - I find it super productive already.

However, it's pretty common to do something like:

        <label>
            Class
            <select name="entry_class" required unicorn:model="entry_class">
                <option value="">-- select --</option>
                {% for comp_class in comp_classes %}
                    <option value="{{ comp_class }}">{{ comp_class }}</option>
                {% endfor %}
            </select>
        </label>

Note the `-- select --` option.

The browser will show it for a second, then it flashes and goes away.

This doesn't seem possible today to have a default option? I've also tried converting my query set to a raw `list()` and inserting at 0 index the `-- select --` option, but then all the options in the dropdown turn into JSON strings after you select something.

I've also tried

`<option value="{{empty_option}}">-- select --</option>`

and then

```
self.empty_option = Something()
```

But this doesn't work either.

Thoughts?

Thanks

**Comment by winrid** [comment]
Ah, okay. The issue is I set the model value to none:

```
entry_class: CompClass | None = None
```

For some reason, None does not match:

```
<option selected>-- select --</option>
```

But this does:

```
entry_class: CompClass
```

I think this is probably a bug (or at least confusing) :)

**Comment by adamghill** [comment]
That does seem confusing! Thanks for reporting this, let me see if I can figure out what might be going on here.

**Comment by winrid** [comment]
Thanks Adam! If my project ever makes money I will definitely donate some small % of profits. Thanks for your work!

**Comment by adamghill** [comment]
Just added a fix for this in https://github.com/adamghill/django-unicorn/pull/563. It will be included in 0.53.0. 👍

**Comment by adamghill** [comment]
0.53.0 has just been published to pypi. 😄
